tests/DotNetLightning.Core.Tests/PeerChannelEncryptorTests.fs

```diff
@@ -14,7 +14,7 @@ let getOutBoundPeerForInitiatorTestVectors () =
 
     let outboundPeer =
         let ie = (Key(hex.DecodeData("1212121212121212121212121212121212121212121212121212121212121212")))
-        PeerChannelEncryptor.newOutBound(NodeId theirNodeId)
+        PeerChannelEncryptor.newOutBound(NodeId theirNodeId, Key ())
         |> Optic.set PeerChannelEncryptor.OutBoundIE_ ie
```

Comment:
I think this line has become useless now
also. would you pass the variable ie here instead of creating new key?